eras/shelley/impl/src/Cardano/Ledger/Shelley/PParams.hs

```diff
@@ -524,10 +525,26 @@ updatePParams :: EraPParams era => PParams era -> PParamsUpdate era -> PParams e
 updatePParams = applyPPUpdates
 {-# DEPRECATED updatePParams "Use applyPPUpdates instead" #-}
 
-pvCanFollow :: ProtVer -> StrictMaybe ProtVer -> Bool
-pvCanFollow _ SNothing = True
-pvCanFollow (ProtVer m n) (SJust (ProtVer m' n')) =
-  (succVersion m, 0) == (Just m', n') || (m, n + 1) == (m', n')
+-- | Check whether the new protocol version is a legitimate version bump with respect to the
+-- previous one.
+pvCanFollow ::
+  -- | Previous protocol version
+  ProtVer ->
+  -- | New protocol version
+  ProtVer ->
+  Bool
+pvCanFollow (ProtVer curMajor curMinor) (ProtVer newMajor newMinor) =
+  (succVersion curMajor, 0) == (Just newMajor, newMinor)
+    || (curMajor, curMinor + 1) == (newMajor, newMinor)
+
+-- | Check whether PParamsUpdate contains a valid `ProtVer` update. No protocol version
```

Comment:
Just an amusing thing (not worth a rebase though) - maybe "Empty protocol version update" would be a little better than "No protocol version" -  At first pass I read it as in: there is no protocol version update that is considered a legal update :laughing: 
But of course it's very clear from the code

Reply:
I had to rebase on master any ways, so I decided to fixup the haddock. I changed it to:

>  When a protocol version update is not included in `PParamsUpdate` it is considered a legal update.

Thank you for pointing out the ambiguity.